### DISCUSSION TITLE
Can a 3rd action button be added?

### DISCUSSION THREAD

**Discussion by SarahYiskah** [mmazzarolo/react-native-modal-datetime-picker#670] [Ideas]
I want to add a button to 'Clear' the date if a date is already selected. Is there a way to do that now?

**Comment by mmazzarolo** [comment]
Hi! If you mean within the picker itself, it's not possible because the code that powers it is native and not in control of this repo :/

**Reply by SarahYiskah** [reply]
Ok, I'll find another way. Thanks for your reply :)
